spec/simplecov_spec.rb

```diff
@@ -0,0 +1,110 @@
+require "helper"
+
+if SimpleCov.usable?
+  describe SimpleCov do
+    describe ".result" do
+      before do
+        SimpleCov.clear_result!
+        allow(Coverage).to receive(:result).once.and_return({})
+        allow(SimpleCov).to receive(:add_not_loaded_files).once.and_return({})
+      end
+
+      context "with merging disabled" do
+        before do
+          allow(SimpleCov).to receive(:use_merging).once.and_return(false)
```

Comment:
i suppose, if we set it back. this feels safer IMO, since we're not changing instance variables of the SimpleCov class, which may have side effects in later specs